src/openforms/api/validators.py

```diff
@@ -10,21 +10,24 @@
 from .utils import get_model_serializer_instance
 
 
-class AllOrNoneRequiredFieldsValidator:
+class AllOrNoneTruthyFieldsValidator:
```

Comment:
I went the wrong way refactoring this and then the functional tests for the confirmation email template broke, so decided to rename the serializer class instead in hopes of making it clearer what the intent is.